packages/flame_isolate/README.md

```diff
@@ -0,0 +1,46 @@
+The power of [integral_isolates](https://pub.dev/packages/integral_isolates) in your
+[Flame](https://pub.dev/packages/flame) game.
+
+
+# Usage
+
+Just add the mixin `FlameIsolate` to your component and start utilizing the power of an isolate as
+simple as running the [compute](https://api.flutter.dev/flutter/foundation/compute-constant.html)
+function.
+
+Example:
+
+```dart
+class MyGame extends FlameGame with FlameIsolate {
+  ...
+  @override
+  void update(double dt) {
+    if (shouldRecalculate) {
+      isolate(recalculateWorld, worldData).then(updateWorld);
+    }
+    ...
+  }
+  ...
+}
+```
+
+
+## Backpressure Strategies
```

Comment:
When you add more info about the backpressure strategies for the docs that info could be copied over here too.